roles/openshift_health_checker/openshift_checks/fluentd_config.py

```diff
@@ -0,0 +1,190 @@
+"""
+Module for performing checks on an Fluentd logging deployment configuration
+"""
+
+import os
+import json
+
+from openshift_checks import OpenShiftCheck, get_var, OpenShiftCheckException
+
+
+class FluentdConfig(OpenShiftCheck):
+    """Module that checks logging configuration of an integrated logging Fluentd deployment"""
+    name = "fluentd_config"
+    tags = ["health"]
+
+    logging_namespace = None
+
+    @classmethod
+    def is_active(cls, task_vars):
+        return super(FluentdConfig, cls).is_active(task_vars)
+
+    def run(self, tmp, task_vars):
+        """Check that Fluentd has running pods, and that its logging config matches Docker's logging config."""
+
+        self.logging_namespace = get_var(task_vars, "openshift_logging_namespace", default="logging")
+        config_error = self.check_logging_config(task_vars)
+        if config_error:
+            msg = ("The following Fluentd logging configuration problem was found:"
+                   "\n-------\n"
+                   "{}".format(config_error))
+            return {"failed": True, "changed": False, "msg": msg}
+
+        return {"failed": False, "changed": False, "msg": 'No problems found with Fluentd logging configuration.'}
```

Comment:
I suspect on '-v' output it will appear in the middle of a big json object?
I'd rather have less noise. `failed: False` conveys the same info.

I have also mixed feelings about the "changed" key. I don't feel confident we use it consistently and correctly, and we don't really use it in any meaningful way. I remember I wanted to "be prepared" for supporting check-mode but thus far I feel we're only carrying extra code.